docs/release-notes/release-notes-0.18.3.md

```diff
@@ -262,6 +266,9 @@ that validate `ChannelAnnouncement` messages.
   our health checker to correctly shut down LND if network partitioning occurs
   towards the etcd cluster.
 
+* [Fix](https://github.com/lightningnetwork/lnd/pull/9050) some inconsintencies
```

Comment:
```suggestion
* [Fix](https://github.com/lightningnetwork/lnd/pull/9050) some inconsistencies
```

Reply:
For some reason your signature disappeared after I pulled and squashed the commits, but feel free to co-sign the commit.